messages/en.json

```diff
@@ -671,7 +670,6 @@
     "succubus_target_success": "You are entrancing \u0002{0}\u0002 tonight.",
     "no_kill_succubus": "You discover that \u0002{0}\u0002 is a succubus and have retracted your kill as a result.",
     "no_target_succubus": "You discover that \u0002{0}\u0002 is a succubus and must now target someone else.",
-    "drunk_target": " In your drunken stupor, you have selected \u0002{0}\u0002 as your target.",
```

Comment:
?

Reply:
Weird, I thought I'd removed that in my drunk branch. Either way it's not used (the full message is used instead)